packages/commons/src/BundledAxeScriptProvider.cs

```diff
@@ -1,8 +1,20 @@
+using System.IO;
```

Comment:
It looks like about half the files marked as changed in this PR are actually unmerged copies of some changes from #57 and #54 that are already in develop; could you please merge develop into the PR branch (or rebase the PR branch onto develop) to remove some of the noise?